Source/CombatExtended/CombatExtended/Settings.cs

```diff
@@ -18,12 +18,14 @@ public class Settings : ModSettings
         private bool allowMeleeHunting = false;
         private bool smokeEffects = true;
         private bool mergeExplosions = true;
+	private bool turretsBreakShields = true;
```

Comment:
Please keep tabs/spaces consistent, at least within the same file (change tabs->spaces please)